tests/api/testcl.hpp

```diff
@@ -507,15 +504,18 @@ class WithCommandQueue : public WithContext {
     cl_device_id device() const { return gDevice; }
 
     void SetUpQueue(cl_command_queue_properties properties) {
+
 #ifndef COMPILER_AVAILABLE
         GTEST_SKIP();
 #endif
-        WithContext::SetUp();
         auto queue = CreateCommandQueue(device(), properties);
         m_queue = queue.release();
     }
 
-    void SetUp() override { SetUpQueue(0); }
+    void SetUp() override {
```

Comment:
Let's create a `WithCommandQueue::SetUpWithQueueProperties` to avoid having to duplicate what is done here in `WithProfiledQueue::SetUp`.